tests/integration/components/bs-alert-test.js

```diff
@@ -88,6 +89,36 @@ module('Integration | Component | bs-alert', function (hooks) {
     assert.ok(action.calledOnce, 'onDismissed has been called.');
   });
 
+  test('When fade is true, onDismissed is called once when visible changes from true to false', async function (assert) {
+    let action = sinon.spy();
+    this.actions.dismissed = action;
+    this.isAlertShown = true;
+
+    await render(
+      hbs`<BsAlert @fade={{true}} @onDismissed={{action "dismissed"}} @visible={{this.isAlertShown}}>Test</BsAlert>`
```

Comment:
For readability and as it's more future proof I prefer this syntax to assign actions in tests:

```suggestion
    this.dismissed = action;
    this.isAlertShown = true;

    await render(
      hbs`<BsAlert @fade={{true}} @onDismissed={{this.dismissed}} @visible={{this.isAlertShown}}>Test</BsAlert>`
```

But that's only about coding style. Feel free to ignore.

Reply:
I like the suggested syntax and will update it, thanks.